app/client/components/identity/EmailAndMarketing/NewsletterSection.tsx

```diff
@@ -46,22 +36,37 @@ const newsletterPreference = (
 	);
 };
 
+function notEmpty<T>(value: T | undefined): value is T {
+	return value !== undefined;
+}
+
+function getColor(theme: string): string {
+	const colors: { [T in Theme]: string } = {
+		[Theme.news]: palette.red.medium,
+		[Theme.opinion]: palette.orange.medium,
+		[Theme.features]: palette.neutral['1'],
+		[Theme.sport]: palette.blue.medium,
+		[Theme.culture]: '#a1845c',
+		[Theme.lifestyle]: palette.pink.medium,
+		[Theme.comment]: '#e05e00',
+		[Theme.work]: palette.neutral['1'],
+		[Theme.fromThePapers]: palette.neutral['1'],
+	};
+
+	if (theme in Theme) {
+		return colors[theme as Theme];
+	}
+	return palette.neutral['1'];
+}
```

Comment:
I think that's a great aim for where to end up. It would also be good to use the Accordion component from Source, but that change was a bit deeper than I had time to make.
I will investigate if it's possible to use the colours from Source in the meantime!